### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by danielnilsson9** [danielnilsson9/bbs-fw#28] [Announcements]
Initial public release of bbshd-fw.

<hr /><em>This discussion was created from the release <a href='https://github.com/danielnilsson9/bbshd-fw/releases/tag/'>v1.0.0</a>.</em>

**Comment by Nobeernogman** [comment]
Hi! 
1) Is there still a throttle issue, or is this fixed?

2) The faq says: There is no original firmware for controller revision 1.4 available
so if you flash this firmware on controller revision 1.4 there is no turning back.

But there is firmware for the 1.4 controller.  Should I send it?

**Reply by danielnilsson9** [reply]
1. Yes, found it, it is fixed! This firmware is working nicely now!

2. Okay, I didn't know, yes please send or post a link here.

**Comment by Nobeernogman** [comment]
I flashed those file's on my old 1.4 fw bbhd controller. 
[CRR10E4830E104001.1.hex](https://mega.nz/file/HF1G3DjZ#sylxqGS7rWNZUTll48EURXnF0LkQXDFCejiXo71v4cU)
[CRR10E5228E104022.1_190418-48TO52V.hex](https://mega.nz/file/PUVRQYQK#iFpkrP131qOpA3-7--Ga6642MeQGaODJiR3jCr3oH8s)

I have downloaded those two files from https://goldenmotor.bike/downloads/ you will see "Convert bbshd from 48v to 52v"
The firmware works with the STC15W4K56S4 controller.

Nice to hear the throttle issue is fixed! I flashed your custom fw on my STC15W4K56S4 bbshd controller months ago, but riding with a throttle issue is nog safe :-D 
Now i have a new 1.5 controller, and i will test your custom fw again!

I gonna donate you a beer!
**-Edit Donated!**

**Reply by danielnilsson9** [reply]
Thank you very much! I can confirm that those firmware files does indeed work on 1.4! I will save them in case anyone may need them in the future.

Glad to hear, please do! And let me know if you find any issues.

Thank you, much appreciated!
